test/Microsoft.TestPlatform.PerformanceTests/Microsoft.TestPlatform.PerformanceTests.csproj

```diff
@@ -8,13 +8,12 @@
   <Import Project="$(TestPlatformRoot)scripts/build/TestPlatform.Settings.targets" />
   <PropertyGroup>
     <OutputType Condition=" '$(TargetFramework)' == 'net6.0' ">Exe</OutputType>
```

Comment:
Looks like this condition is no longer needed (you removed net6.0).

Reply:
I am pretty sure I would forget to put it back when I make this work on .NET, if ever. So let's just keep it.